doc/configuration-v2_2-experimental.md

```diff
@@ -29,11 +29,13 @@ The Ignition configuration is a JSON document conforming to the following specif
       * **_start_** (integer): the start of the partition (in device logical sectors). If zero, the partition will be positioned at the earliest available part of the disk.
       * **_typeGuid_** (string): the GPT [partition type GUID][part-types]. If omitted, the default will be 0FC63DAF-8483-4772-8E79-3D69D8477DE4 (Linux filesystem data).
       * **_guid_** (string): the GPT unique partition GUID.
+      * **_wipePartition_** (boolean) whether or not to delete the partition before creation. TODO(ajeddeloh): fix docs to have filesystem, partition, and raid reuse semantics instead of just filesystem
```

Comment:
Are we erasing in place with filesystems? I think a better term for what we're doing with filesystems is overwriting them. `reuseOldPartitionNumber` would be more accurate but is a terrible name. `force` is vague and inaccurate. Maybe `wipePartitionEntry`?

Reply:
Overwriting, yes.

`wipePartitionEntry` isn't bad.  `recreatePartition`?

Reply:
As per [my earlier comment](https://github.com/coreos/ignition/pull/436#discussion_r137618256), what this really does is control whether Ignition is allowed to delete partition entries to achieve it's goal state. Maybe something to reflect that? `permitEntryDeletion`? On the other hand, it will _always_ delete a partition if specified and that doesn't make that clear. `deletePartitionEntry`? 

I don't like `recreatePartition` because we may be using it to just delete a partition.

Reply:
Given `shouldExist` (and I'd like to find a better name for that too), `wipePartitionEntry` seems like the least ambiguous so far.